### ISSUE TITLE
Make it a UWP app - win10

### ISSUE THREAD

**Issue by dutch2005** [matthinc/HomeAssistantElectron#18]
Any chance you can make this a full Windows10 Store application? So it also always is the latest version (if there are any updates and the sorts?

cool program, keep it up :)

**Comment by matthinc** [comment]
Maybe sometimes...
The main problem is that this would be a lot of work. Creating a store account, publish every release... Then the Mac-users would want to see it in the MacAppstore etc etc. 

This is really just a side project for me and I don't have too much spare time between work and study.
And I would rather use this time to add new features. But never say never. I will keep this open as a reminder to myself. Maybe in the next semester holidays ;)

**Comment by dutch2005** [comment]
I did some digging into this and technically everyone can do this for themselves.

How? Well instead of going to https://yourwebsite.xyz (or https://1.2.3.4 you could...

1) https://www.pwabuilder.com/
2) Enter your own dynamic hostname / website-name you've given it
3) the site will make an application for Windows10/Win10M, Win10 holo, Android and iOS

Big downside atm is that it would need a fixed site (and everyone has their own website), the apps itself could then possible to edited cose wise so that folks can enter their own information in it e.g. hostname and the optional password.

Basically since HomeAssistent is 90% website being able to make a wrapper over it would be nice, heck they even offer (via commandline) offer to make it compatible with win32 thus including Windows 7 + Windows 8/8.1 Users, potentiall even Linux (via Wine)
